lib/checks/001-deprecations.js

```diff
@@ -21,6 +21,11 @@ checkDeprecations = function checkDeprecations(theme, themePath) {
             ruleCode: 'GS001-DEPR-IMG'
         },
         {
+            helperRegEx: /{{\s*?cover\b[\w\s='"]*?}}/g,
```

Comment:
Also we want to have a conditional check as themes are quite likely to use `{{# if cover}}` e.g `/{{\s*?#if\s*?cover\s*?}}/g`